src/bin/directoryd.rs

```diff
@@ -27,5 +27,5 @@ fn main() {
 
     let directory = Arc::new(DirectoryServer::new(args.data_directory, Some(conn_type)).unwrap());
 
-    start_directory_server(directory);
+    start_directory_server(directory).expect("Error in starting directory server.");
```

Comment:
clippy doesn't let me commit without removing it. I can resolve the merge conflicts, while merging the PR. but I believe I won't have the permissions for that?

So, shall I use --no-verify flag and compile the code? But this might make other things vulnerable.

As of now, I've addressed a few of your comments in https://github.com/citadel-tech/coinswap/pull/269/commits/bb7d3be1030bad2dcdb5eaafe4d1d7a529ac1dfb